### PR TITLE
chore: update vite to 5.1.1

### LINKED ISSUE #474: Update Vite v5.1
I tried to upgrade Vite 5.1.1 in https://github.com/dai-shi/waku/pull/471, but it broke something related to styles.

### PR DESCRIPTION
**Pull request by Aslemammad**
Resolves #474

### CODE REVIEW

**Comment by Aslemammad** [comment]
locally it works, but there should be something weird going on in the CI

File: e2e/fixtures/rsc-basic/package.json [hunk 1]
```diff
@@ -12,6 +12,7 @@
     "react": "18.3.0-canary-4b2a1115a-20240202",
     "react-dom": "18.3.0-canary-4b2a1115a-20240202",
     "react-server-dom-webpack": "18.3.0-canary-4b2a1115a-20240202",
+    "vite": "^5.1.1",
```
Comment (dai-shi) [inline e2e/fixtures/rsc-basic/package.json:15]: Why adding this?

**Comment by dai-shi** [comment]
> locally it works, but there should be something weird going on in the CI

It's the other way around. It fails with the standalone usage. It would be very nice if we can reproduce the bug in the monorepo setup.

CI test is added to make it sure, but reproduction should be done locally in advance.

@Aslemammad Sorry, I didn't write my findings there. Let me know if anything is unclear.

**Comment by dai-shi** [comment]
Confirmed https://pkg.csb.dev/dai-shi/waku/commit/a57ea3ab/waku works fine with standalone 01_template.

**Review by dai-shi — APPROVED** [review]
The diff looks good to me.

**Pull request merged**